packages/bridging/src/providers/near-intents/const/index.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  arbitrumOne,
+  avalanche,
+  base,
+  bnb,
+  ChainInfo,
+  gnosisChain,
+  mainnet,
+  optimism,
+  polygon,
+} from '@cowprotocol/sdk-config'
+import { HOOK_DAPP_BRIDGE_PROVIDER_PREFIX } from '../../../const'
+import type { Hex } from 'viem'
+import { BridgeStatus } from '../../../types'
+
+export const NEAR_INTENTS_HOOK_DAPP_ID = `${HOOK_DAPP_BRIDGE_PROVIDER_PREFIX}/near-intents`
+
+export const NEAR_INTENTS_SUPPORTED_NETWORKS = [
+  arbitrumOne,
+  avalanche,
+  base,
+  bnb,
+  gnosisChain,
+  mainnet,
+  optimism,
+  polygon,
+]
```

Comment:
Would it be simpler to map just the names to the chain ids? So using `ChainId` type, instead of the full object. 

You can always use later `getChainInfo` to get the details

This applies also to `NEAR_INTENTS_BLOCKCHAIN_TO_COW_NETWORK`